protocol/solid-protocol-test-manifest.ttl

```diff
@@ -27,28 +27,28 @@ manifest:authentication-header
   a td:TestCase ;
     spec:requirementReference sopr:server-authentication,
     sopr:server-unauthenticated ;
-  td:reviewStatus td:unreviewed ;
+  td:reviewStatus td:approved ;
   spec:testScript
     <https://github.com/solid/specification-tests/protocol/authentication/header.feature> .
```

Comment:
@RubenVerborgh It's great to have some feedback on the tests. @kjetil and I have a task to review the style of the tests created so far and this is the sort of feedback that will help with that review. 

Regarding the test container - since this is how every test is set up we need to make it clearer in the documentation (and perhaps think about function names) but every test container is given a unique identifier using UUIDv4. We are planning a change to come up with a schema for shorter identifiers so the URLs are not so difficult to read. The README describes these functions as:
> #### `createTestContainer()`
> Create a SolidContainer object referencing a unique sub-container of the `rootTestContainer`.
  This container will not be created until a resource is created inside it.
>
> #### `createTestContainerImmediate()`
> Create a SolidContainer object referencing a unique sub-container of the
  `rootTestContainer`, but ensure that it is actually created at this point.

For this and other suggestions, I've created a separate issue: https://github.com/solid/specification-tests/issues/44

Reply:
> `match header WWW-Authenticate != null` (why) is that sufficient? Do we want/need more detail?

Just to comment on that case: That's all that is required from the RFC in context of Solid Protocol. Then, authn protocols will have more detailed requirements that needs to be tested in the context of that protocol.